CPP-021/cpp-021.xml

```diff
@@ -0,0 +1,625 @@
+                        AIP or class of AIPs that is adequate for long-term 
+                        preservation, enabling the identification, extraction 
+                        and interpretation of all the required components within 
+                        each AIP.</li>
+                    <li>4.3.5 The repository must be able to export the relevant 
+                        AIPs to its successor location in the timescale required, 
+                        together with supplementary information including 
+                        specifications of the AIPs adequate for the information 
+                        contained in them to be extracted.</li>
+                    <li>4.4.1.1 The repository shall preserve the Content 
+                        Information of AIPs</li>
+                    <li>5.1.2 The repository shall manage the number, security, 
+                        coordination, and location of copies of all digital 
+                        objects.</li>
+                </ul>
+            </cpp:correspondingSection>
+        </cpp:mapping>
+        <cpp:mapping>
+            <cpp:frameworkName>OAIS</cpp:frameworkName>
+            <cpp:correspondingTerm>
+                <p>AIP Version</p>
+                <p>AIP Edition</p>
+            </cpp:correspondingTerm>
+            <cpp:correspondingSection>
+                <p>Section 5.2.4.6 "Distinguishing AIP Versions, AIP Editions 
+                    and Derived AIPs"</p>
+                <p>OAIS distinguishes AIP Versions (triggered by migrations 
+                    performed by the Archive) and AIP Editions (triggered by 
+                    update requests from the Producer).</p>
+            </cpp:correspondingSection>
+        </cpp:mapping>
+        <cpp:mapping>
+            <cpp:frameworkName>PREMIS</cpp:frameworkName>
+            <cpp:correspondingSection>
+                <p>PREMIS does not approach versioning. Nevertheless, the "1:1 
+                    principle" implies that if an Object (Representation, File 
+                    or Bitstream) is modified by the repository, this operation 
+                    creates a new Object that is declared as being derived from 
+                    the source Object.</p>
+            </cpp:correspondingSection>
+        </cpp:mapping>
+    </cpp:frameworkMappings>
+    <cpp:referenceImplementations>
+        <cpp:useCases>
+            <cpp:useCase>
+                <cpp:useCasetitle>BnF Information Package lifecycle Implementation</cpp:useCasetitle>
+                <cpp:institution>
+                    <cpp:institutionLabel>Bibliothèque nationale de France</cpp:institutionLabel>
+                    <cpp:institutionCountry>FR</cpp:institutionCountry>
+                </cpp:institution>
+                <cpp:linkToDocumentation xml:lang="en">
+                    <cpp:hyperlink>
+                        https://bnf.hal.science/hal-01617645v1</cpp:hyperlink>
```

Comment:
I think that having just the link is OK - though I always recommend adding descriptive metadata in a citation, and not only an identifier, as identifiers can break... ;-)